Duplicated need for `yarn install` command

**Describe the bug**
Since the Date Picker component was added, we have to run `yarn install` twice (one for the `root` folder and one for the `react` folder.

This is due to the `date-fns` dependency. This package is needed for packages inside @vtex/react-scripts (version 1.29.0) and inside react-datepicker (version 2.0.0-alpha.26), as we can see in the image below:

![image](https://user-images.githubusercontent.com/5971264/50175639-7b8abe00-02e4-11e9-84ca-278fb042c133.png)
 
So, when you run the `yarn install` command on the `root` folder the version 1.29.0 is installed and it breaks the react-datepicker package due to:

```
Module not found: Can't resolve 'date-fns/locale/index.js' in '/Users/kevin/repos/styleguide/react/components/DatePicker'
```

**Expected behavior**
There should be no need for running the `install` command twice. If it is really needed, it should be in the development documentation (README, perhaps?)

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Fix
+
+- Fix #458 by adding postinstall script to run the install dependencies for the
+  `react/` folder after installing dependencies for the `root` folder.
+
 ## [8.10.2] - 2019-01-08
 
 ### Added
@@ -16,6 +21,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [8.10.1] - 2019-01-07
 
 ### Changed
+
 - **Toast** Allow single line toast on small screens if the message is short enough.
 
 ## [8.10.0] - 2019-01-07
@@ -42,9 +48,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [8.9.0] - 2018-12-28
 
 ### Added
+
 - **ButtonWithIcon** New component
 
 ### Deprecated
+
 - **Button** `icon` prop was deprecated in favor of the `ButtonWithIcon` component
 
 ## [8.8.1] - 2018-12-28

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [8.10.3] - 2019-01-08
+
 ### Fix
 
 - Fix #458 by adding postinstall script to run the install dependencies for the

--- a/README.md
+++ b/README.md
@@ -70,9 +70,10 @@ yarn deploy
 
 - Your project has to run with webpack >= 2. Here's a [guide](https://webpack.js.org/migrate/3/) for upgrading Webpack to v2.
 
-- `VTEX Styleguide` is a project built to run on top of `npm` or `VTEX IO`. To
-  make this viable we currently use two `package.json`, one in the toor repository
-  folder for `npm` and one inside the `react` folder for `VTEX IO`. We are
-  currently studying how to improve this architecture, but for now we have to live
-  with duplicated dependencies in these `package.json` files (For more info please
-  refer to #458, #459 and #483).
+- `VTEX Styleguide` is a project built to run on top of either `npm` or `VTEX IO`. To
+  make this viable, we currently use two `package.json` files, one in the top repository
+  folder for `npm`, and one inside the `react` folder for `VTEX IO`. We are
+  looking into how we can improve this architecture, but for now we have to live
+  with duplicated dependencies in these `package.json` files.
+
+  For more info, please refer to https://github.com/vtex/styleguide/issues/483

--- a/README.md
+++ b/README.md
@@ -69,3 +69,10 @@ yarn deploy
 #### Known issues
 
 - Your project has to run with webpack >= 2. Here's a [guide](https://webpack.js.org/migrate/3/) for upgrading Webpack to v2.
+
+- `VTEX Styleguide` is a project built to run on top of `npm` or `VTEX IO`. To
+  make this viable we currently use two `package.json`, one in the toor repository
+  folder for `npm` and one inside the `react` folder for `VTEX IO`. We are
+  currently studying how to improve this architecture, but for now we have to live
+  with duplicated dependencies in these `package.json` files (For more info please
+  refer to #458, #459 and #483).

--- a/manifest.json
+++ b/manifest.json
@@ -1,7 +1,7 @@
 {
   "vendor": "vtex",
   "name": "styleguide",
-  "version": "8.10.2",
+  "version": "8.10.3",
   "title": "VTEX Styleguide",
   "description": "The VTEX Styleguide components for the Render framework",
   "builders": {

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@vtex/styleguide",
-  "version": "8.10.2",
+  "version": "8.10.3",
   "scripts": {
     "test": "react-scripts test --env=jsdom",
     "test:codemod": "jest codemod",

--- a/package.json
+++ b/package.json
@@ -20,7 +20,8 @@
     "postreleasy": "vtex publish --public --verbose",
     "lint": "eslint react",
     "lint-fix": "eslint react --fix --max-warnings 0",
-    "prettier-conflict-check": "eslint --print-config .eslintrc | eslint-config-prettier-check"
+    "prettier-conflict-check": "eslint --print-config .eslintrc | eslint-config-prettier-check",
+    "postinstall": "yarn --cwd react --ignore-scripts"
   },
   "files": [
     "lib/",

